src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_buffer_fusing.cpp

```diff
@@ -131,6 +131,7 @@ bool concat_in_place_optimization::match(const program_node& concat_node,
                                   pred_params[0].get_output_layout().data_padding._lower_size[concat_axis]);
 
```

Comment:
random spot)
* could you check crop case as well?
* Maybe should we add some assertion for checking such alignment before calling onednn api?